utils.js

```diff
@@ -121,11 +121,11 @@ module.exports = {
     cmd:
     {
         /**
-        * Check if gulp is running with Administration priviledges
+        * Check if gulp is running with Administration privileges
         * 
         * @param function(boolean) callback Callback function
         */
-        hasAdministrationPriviledges: function (callback) {
+        hasAdministrationPrivileges: function (callback) {
```

Comment:
Humm This is indeed a typo, but creates a breaking change.
Also, I'm not seeing this change in other places, so either it's not used anymore or it's used by an external package that relies on this (I believe we used to use this function on npm.postinstall.js on the repo root). Perhaps it's best not to change this at this time.